frontend/app/src/components/StreamlitDialog/DeployDialog/DeployDialog.tsx

```diff
@@ -192,8 +194,8 @@ export function DeployDialog(props: DeployDialogProps): ReactElement {
             <StyledSubheader>Custom deployment</StyledSubheader>
             <ListElement>For companies</ListElement>
             <ListElement>
-              Deploy on your own hardware or in the cloud, with Docker,
-              Kubernetes, etc
+              Deploy on your hardware or in the cloud, with Docker, Kubernetes,
```

Comment:
note: Removed the `own` so that the buttons of the two dialog columns are aligned with each other again after changing the sizes slightly:

Before:
<img width="982" alt="Screenshot 2024-10-04 at 15 42 06" src="https://github.com/user-attachments/assets/4d0db9d3-7b8c-48c2-af2c-a8c97992ffc9">

After:
<img width="885" alt="Screenshot 2024-10-04 at 15 41 17" src="https://github.com/user-attachments/assets/9cc61ebf-6ed2-44bc-b29c-81c268dccf6e">

Relying on the text content is probably not the greatest, but since the styling was built this way and I don't expect the dialog to be touched again soonish (this was the first time in 16 months), I don't think investing time in a better mechanism makes sense now